docs/README.md

```diff
@@ -112,6 +112,20 @@ You can see full usage with `--help`, here an example:
 python create_images.py -i INPUT_TPSTREAM.hdf5 -n 1000 -o my/output/folder/ --ticks-limit 5 --channel-limit 2 --min-tps 3 
 ```
 
+#### `tpg_find_overactive_channels.py`
+This script generates a histogram for the number of TPs that occur on a given channel. Overly active channels have a TP count that can be considered as an outlier.
+This script also prints the channels that have a TP count above a user given limit and by the outlier definition with [Tukey's upper fence](https://en.wikipedia.org/wiki/Outlier#Tukey's_fences).
+The plot that is saved is unique for a given HDF5 file; however, running over the same HDF5 file with different arguments will overwrite previous plots, so be careful.
+There can also be more than one plot produced if there is more than one detector element (CRP/APA) for the requested detector (HD, VD, VDCB). If no TPs are found for a detector element, plotting is skipped and a print informs you that the detector element was empty.
```

Comment:
Very nice, so we'll get separate masks for each APA in pd2.

Reply:
The motivation was primarily from a problem with the Tukey's fence: if one fragment only has the info of 1 APA, then there are 2560 * 3 zeros that are confounding the statistics.